tests/client.go

```diff
@@ -0,0 +1,40 @@
+package tests
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	v1 "kubevirt.io/api/core/v1"
+
+	networkv1 "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
+)
+
+var clientInstance client.Client
+
+func createClient() (client.Client, error) {
+	scheme := runtime.NewScheme()
+	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
+	utilruntime.Must(v1.AddToScheme(scheme))
+	utilruntime.Must(networkv1.AddToScheme(scheme))
+	ctrlOptions := ctrl.Options{
+		Scheme:             scheme,
+		MetricsBindAddress: "0", // disable metrics
+	}
+	mgr, _ := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrlOptions)
```

Comment:
please check the err here

Reply:
but NewManager might err when GetConfigOrDie doesnt

Reply:
done